decimal/decimal.go

```diff
@@ -0,0 +1,109 @@
+// Package decimal with support of Tarantool's decimal data type.
+//
+// Decimal data type supported in Tarantool since 2.2.
+//
+// Since: 1.6
+//
+// See also:
+//
+// * Tarantool MessagePack extensions https://www.tarantool.io/en/doc/latest/dev_guide/internals/msgpack_extensions/#the-decimal-type
+//
+// * Tarantool data model https://www.tarantool.io/en/doc/latest/book/box/data_model/
+//
+// * Tarantool issue for support decimal type https://github.com/tarantool/tarantool/issues/692
+//
+// * Tarantool module decimal https://www.tarantool.io/en/doc/latest/reference/reference_lua/decimal/
+package decimal
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/shopspring/decimal"
```

Comment:
I'm a bit confused. There is
- a string representation
- BCD which is a bytes array with specific structure
- some external decimal package

How do they correspond?

Reply:
it's BCD (from messagepack) -> string -> external lib